### ISSUE TITLE
Keras - losses

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#566]
Keras model 的loss 函数在 complie model 时定义：
```
model.compile(loss='mean_squared_error', optimizer='sgd')
```
> 此处使用 `MSE`  loss function

# Reference
- [ ] [keras loss docs](https://keras.io/losses/)
- [ ] [Loss code](https://github.com/keras-team/keras/blob/master/keras/losses.py)
- [ ] [keras中的损失函数](https://zhuanlan.zhihu.com/p/34667893) 

# 可以使用的loss function
- [mean_squared_error](https://github.com/keras-team/keras/blob/master/keras/losses.py#L606) - MSE
- mean_absolute_error - MAE
- mean_absolute_percentage_error - MAPE
> 100.0*K.mean(K.abs((y_true - y_pred) / K.clip(K.abs(y_true), K.epsilon(), None)), axis=-1)
- mean_squared_logarithmic_error - MSLE
- squared_hinge
![image](https://user-images.githubusercontent.com/2216970/67673643-85c22200-f9b5-11e9-83a3-84b92652c2ab.png)

- hinge
![image](https://user-images.githubusercontent.com/2216970/67673659-8eb2f380-f9b5-11e9-8392-89cb4f2af92d.png)

- logcosh
- huber_loss
- kullback_leibler_divergence
- poisson
![image](https://user-images.githubusercontent.com/2216970/67673756-d174cb80-f9b5-11e9-927e-c021a1093ddd.png)

- cosine_proximity
![image](https://user-images.githubusercontent.com/2216970/67673773-d9347000-f9b5-11e9-97c6-0db8515bd9e2.png)

---

- categorical_hinge
- **categorical_crossentropy**
- **binary_crossentropy**
- sparse_categorical_crossentropy
- is_categorical_crossentropy

**Comment by junxnone** [comment]
#474 #9
